src/electionguard/publish.py

```diff
@@ -1,20 +1,19 @@
-from typing import List
-from jsons import set_serializer
 from os import mkdir, path
-from jsons import set_serializer, dump
+from typing import List
 
+from jsons import set_serializer, dump, set_deserializer, set_validator
 
 from .ballot_box import CiphertextBallot
 from .decryption_mediator import PlaintextTally
 from .election import CiphertextElectionContext, ElectionConstants, ElectionDescription
+from .group import ElementModP, ElementModQ, int_to_p_unchecked, int_to_q_unchecked
 from .key_ceremony import CoefficientValidationSet
 from .tally import CiphertextTally
-from .group import ElementModP, ElementModQ
 
-RESULTS_DIR = "results" + path.sep
-COEFFICIENTS_DIR = RESULTS_DIR + "coefficients" + path.sep
-BALLOTS_DIR = RESULTS_DIR + "encrypted_ballots" + path.sep
-SPOILED_DIR = RESULTS_DIR + "spoiled_ballots" + path.sep
+RESULTS_DIR = "results"
+COEFFICIENTS_DIR = path.join(RESULTS_DIR, "coefficients")
+BALLOTS_DIR = path.join(RESULTS_DIR, "encrypted_ballots")
+SPOILED_DIR = path.join(RESULTS_DIR, "spoiled_ballots")
```

Comment:
The `path.join` is a good addition but this should likely be an isolated issue and PR since it pertains to the export / serialization of the data and not the import / deserialization.

Reply:
It impacted me because I wasn't putting path separators in my own code.

Reply:
That may be true. It still should be isolated as separate PR.

Reply:
Okay, I'm fine with that. FWIW, I'm depending on this behavior now.

Reply:
Created issue #116